flake.nix

```diff
@@ -42,6 +42,7 @@
           gcc
           rust
           postgresql
+          tesseract4
```

Comment:
Not really needed for operation?

Reply:
Helpful for development as we have discussed.